web/admin/src/Admin.js

```diff
@@ -736,9 +736,13 @@ export default class Admin extends Component {
   }
 
   offAnom = () => {
-    const anom = this.state.anom[0]
-    this.props.fbc.database.public.adminRef('askAnom').child(anom.key).update({"allow": false})
-
+    if (this.state.anom.length === 0) {
```

Comment:
Can you add a comment about when anom.length would be empty vs having at least one element, and an explanation for the `false` value being added at different places in the data tree?